terraform/spec/dependabot/terraform/requirement_spec.rb

```diff
@@ -16,3 +16,17 @@
     end
   end
 end
+
+RSpec.describe Dependabot::Terraform::Requirement do
+  subject(:requirement) { described_class.new(requirement_string) }
+  let(:requirement_string) { ">=v1.0.0" }
+
+  describe ".new" do
+    it { is_expected.to be_a(described_class) }
+
```

Comment:
No need to repeat all of this, we can nest the code in the existing block and it will inherit all of the setup of the previous tests 👍 

```suggestion

```